scripts/download_tdr_parquet.py

```diff
@@ -0,0 +1,103 @@
+"""
+Export parquet files from TDR and download them to local storage.
```

Comment:
```suggestion
Export Parquet files from TDR and download them to local storage.
```

Here and in elsewhere in documentation.